charts/testkube-api/values.yaml

```diff
@@ -15,6 +15,9 @@ global:
 ## Kubernetes version (using Helm capabilities if not set)
 kubeVersion: ""
 
+rbac:
+  create: true
+
```

Comment:
Should installCRD be true by default

Reply:
good point added default everywhere (api don't have installCRD just operator)